js/wee.routes.js

```diff
@@ -184,35 +184,40 @@
 		 * @returns {object} data
 		 */
 		uri: function(value) {
-			if (value) {
-				if (W.$isObject(value)) {
-					uri = W.$extend(this.uri(), value);
-					return uri;
-				}
-
-				var a = W._doc.createElement('a');
-				a.href = value;
-
-				var search = a.search,
-					path = a.pathname.replace(/^\/|\/$/g, '');
-
-				uri = {
-					full: '/' + path + search + a.hash,
-					hash: a.hash.slice(1),
-					history: false,
-					path: '/' + path,
-					query: search ? W.$unserialize(search) : {},
-					segments: path.split('/'),
-					url: a.href
-				};
-			} else if (! uri) {
-				uri = W.routes.uri(location.href);
+			if (! value && uri) {
```

Comment:
I see... the updating you are speaking of is this in `_process`...

```
if (W.routes) {
				// Update current path
				W.routes.uri(conf.path);
...
```

This is what you were talking about?